.github/workflows/functional_tests.yaml

```diff
@@ -85,7 +85,8 @@ jobs:
 
           # Delete unit tests, so we can get to testing faster
           sed -i '/tests\/unit/d' mozillavpn.pro
-          qmake CONFIG+=DUMMY QMAKE_CXXFLAGS+=--coverage QMAKE_LFLAGS+=--coverage CONFIG+=debug CONFIG+=inspector QT+=svg
+          qmake QMAKE_CXX=g++-10 QMAKE_LINK=g++-10 QMAKE_CXXFLAGS+=--coverage QMAKE_LFLAGS+=--coverage \
```

Comment:
Can you also change the grcov failure workaround in the GitHub action?

Reply:
Unfortunately, bumping the compiler to GCC-10 didn't actually solve the issue with our github workflow. I think this is something that we actually need to fix in the grcov project instead.